### PR TITLE
Package Structure refactor 

### PR DESCRIPTION
**Pull request by stevenjob**
Now use a multi project gradle build.

### CODE REVIEW

**Comment by jdhodges** [comment]
This seems like a good start with our alification process

File: backend/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+build/
```
Comment (rjmcneill) [inline backend/.gitignore]: new line

File: build.gradle [hunk 2]
```diff
@@ -1,68 +1,2 @@
...
-    dependsOn build
-}
-
-//This task runs webpack to bundle the frontend to the package folder
-task runWebpack(type: NpmTask) {
-    args = ["run", "bundle"]
-}
-bootRun.dependsOn runWebpack
-runWebpack.dependsOn npm_install
+version = '0.2.0'
```
Comment (rjmcneill) [inline build.gradle]: new line

File: frontend/.gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+dist/
```
Comment (rjmcneill) [inline frontend/.gitignore]: new line

File: frontend/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "tweet-scuttler",
-  "version": "0.0.1",
+  "version": "0.2.0",
```
Comment (rjmcneill) [inline frontend/package.json]: 0.1.0?

**Comment by rjmcneill** [comment]
:+1:

File: frontend/package.json (hunk 4 above)
Comment (stevenjob) [inline frontend/package.json]: 0.1 was the POC release

File: frontend/.gitignore (hunk 3 above)
Comment (stevenjob) [inline frontend/.gitignore]: cheers m9

File: frontend/package.json (hunk 4 above)
Comment (rjmcneill) [inline frontend/package.json]: So are we not building upon that now?
So 0.1.1?

Comment (stevenjob) [inline frontend/package.json]: aye done

**Comment by jdhodges** [comment]
LGTM, merged :+1:

**Pull request merged**